test/api/link.js

```diff
@@ -0,0 +1,51 @@
+/*!
+ * test/api/link.js
+ * 
+ * Copyright (c) 2014
+ */
+
+// Core
+var net = require('net');
+
+/* -----------------------------------------------------------------------------
+ * process piping
+ * ---------------------------------------------------------------------------*/
+
+var stdin = process.stdin,
+    inputChunks = [],
+    opts = {};
```

Comment:
This is not required (also not documented anywhere).

I prefer not to use comma separator to define multiple variables.

1. If I wanted to add a a var under opts, I would have to change 2 lines rather than 1. This shows up in git diffs and thus included in Code Reviews etc. Its not a huge deal, but you added an extra line that needs to be reviewed for really no additional benefit.

2. Commenting out blocks of code. If I wanted to comment out `opts` for some reason (it happens I swear), I would have to comment it out and also alter `inputChunks` to end with a `;`.

My personal preference is to just right the entire definition:

```
var stdin = process.stdin;
var inputChunks = [];
var opts = {};
```